src/content/en/2019/security.md

```diff
@@ -0,0 +1,460 @@
+| mobile  | TLS 1.0     | 623695   | 90232813 | 0.69  |
+| desktop | TLS 1.0     | 616926   | 79586221 | 0.78  |
+| mobile  | TLS 1.1     | 9727     | 90232813 | 0.01  |
+| desktop | TLS 1.1     | 7481     | 79586221 | 0.01  |
+
+### Certificate Authorities 
+Of course, if we want to use HTTPS on our website then we need a certificate from a Certificate Authority. With the increase in the use of HTTPS comes the increase in use of CAs and their products/services. Here are the top 10 certificate issuers based on the volume of TLS requests that use their certificate.
+
+<br/>
+
+| client  | issuer                                 | requests | total    | pct   |
+|---------|----------------------------------------|----------|----------|-------|
+| mobile  | Google Internet Authority G3           | 15177834 | 77109521 | 19.68 |
+| desktop | Google Internet Authority G3           | 10760946 | 55866534 | 19.26 |
+| desktop | Let's Encrypt Authority X3             | 5698524  | 55866534 | 10.2  |
+| desktop | DigiCert SHA2 High Assurance Server CA | 5491199  | 55866534 | 9.83  |
+| mobile  | DigiCert SHA2 High Assurance Server CA | 7141087  | 77109521 | 9.26  |
+| mobile  | Let's Encrypt Authority X3             | 7085319  | 77109521 | 9.19  |
+| mobile  | DigiCert SHA2 Secure Server CA         | 6727708  | 77109521 | 8.72  |
+| mobile  | GTS CA 1O1                             | 6497589  | 77109521 | 8.43  |
+| desktop | GTS CA 1O1                             | 4395585  | 55866534 | 7.87  |
+| desktop | DigiCert SHA2 Secure Server CA         | 4217289  | 55866534 | 7.55  |
+
+<br/>
+
+The rise of Let's Encrypt has been meteoric after their launch in early 2016, since then they've become one of the top certificate issuers in the world. The availability of free certificates and automated tooling has been critically important to the adoption of HTTPS on the Web and Let's Encrypt certainly had a significant part to play in both of those.
+
+### Authentication key type
+Alongside the important requirement to use HTTPS is the requirement to also use a good configuration. With so many configuration options and choices to make, this is a careful balance. Looking first of all the keys used for authentication purposes, we still see a large % of the Web using RSA.
+
+<br/>
+
+| client  | is_ecdsa | is_rsa   | total    | pct_ecdsa | pct_rsa |
+|---------|----------|----------|----------|-----------|---------|
+| mobile  | 23832006 | 53056268 | 90232714 | 26.41     | 58.8    |
+| desktop | 17090634 | 38734846 | 79586141 | 21.47     | 48.67   |
+
+<br/>
+
+Whilst ECDSA keys are stronger and demonstrate better performance than their RSA counterparts, concerns around backwards compatibility do prevent some website operators from migrating. 
+
+### Forward Secrecy
+The importance of Forward Secrecy is now well understood. It was introduced as an optional configuration in 2008 with TLSv1.2 and has become mandatory in 2018 with TLSv1.3 requiring it.
```

Comment:
Worth explaining briefly what forward secrecy is? e.g.

```suggestion
The importance of Forward Secrecy, a property of some key exchange algorithms that prevents each connection to a server from being exposed in case of a compromise of the server certificate's private key, is now well understood. It was introduced as an optional configuration in 2008 with TLSv1.2 and has become mandatory in 2018 with TLSv1.3 requiring it.
```